rs/cli/src/runner.rs

```diff
@@ -444,4 +445,30 @@ impl Runner {
             .await?;
         Ok(())
     }
+
+    pub async fn network_heal(
+        &self,
+        request: ic_management_types::requests::HealRequest,
+        _verbose: bool,
+        simulate: bool,
+    ) -> Result<(), anyhow::Error> {
+        let change = self.dashboard_backend_client.network_heal(request).await?;
+        println!("{}", change);
+
+        join_all(change.subnets_change_response.iter().map(|subnet_change_response| async move {
+            self.run_membership_change(
+                subnet_change_response.clone(),
+                ops_subnet_node_replace::replace_proposal_options(subnet_change_response)?,
+                simulate,
+            )
+            .await
+            .map_err(|e| {
+                println!("{}", e);
+                e
+            })
+        }))
+        .await;
```

Comment:
```suggestion
       let errors = join_all(change.subnets_change_response.iter().map(|subnet_change_response| async move {
            self.run_membership_change(
                subnet_change_response.clone(),
                ops_subnet_node_replace::replace_proposal_options(subnet_change_response)?,
                simulate,
            )
            .await
            .map_err(|e| {
                println!("{}", e);
                e
            })
        }))
        .await
        .into_iter()
        .filter_map(|f| f.err())
        .collect::<Vec<_>>();

        if !errors.is_empty() {
            anyhow::bail!("Errors: {:?}", errors);
        }
```

Reply:
For consistency, I would skip a `println!` here and just bail on errors and print them as a return of the whole command